recipes/archer/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: archer
+  version: {{ version }}
+
+source:
+  url: https://github.com/will-rowe/archer/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 34390e947644ae0b239ed4fae7b74a069a11c87e4fd575e19cf3aff2cf89800b
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Does it still need a `c` compiler if `CGO_ENABLED=0` is set.

Reply:
You're right - I can remove that.